plugin/hooks.go

```diff
@@ -35,6 +35,7 @@ const (
 	UserHasBeenCreatedID     = 17
 	ReactionHasBeenAddedID   = 18
 	ReactionHasBeenRemovedID = 19
+	OnPluginEventID          = 20
```

Comment:
I think this is misleading, I think should be `OnPluginClusterEventID`

Reply:
As usual naming was one of the hardest part of the design and I was trying to keep it short but I agree that this way it can read ambiguous/unclear. I'll update.